Client/Experiments/initial_experiments.json

```diff
@@ -1,216 +1,57 @@
-                                    "libraryShortcuts": true
-                                }
-                             },
-                            "enabled": true,
-                            "featureId": "homescreen"
-                        }
-                    },
-                    {
-                        "slug": "distraction-free",
-                        "ratio": 0,
-                        "feature": {
-                            "value": {
-                                "sections-enabled": {
-                                    "topSites": false,
-                                    "jumpBackIn": false,
-                                    "recentlySaved": false,
-                                    "pocket": false,
-                                    "libraryShortcuts": false
-                                }
-                             },
-                            "enabled": true,
-                            "featureId": "homescreen"
-                        }
-                    }
-                ],
-                "outcomes": [],
-                "arguments": {},
-                "probeSets": [],
-                "startDate": null,
-                "targeting": "true",
-                "featureIds": [
-                    "homescreen"
-                ],
-                "application": "org.mozilla.firefox_beta",
-                "bucketConfig": {
-                    "count": 10000,
-                    "start": 0,
-                    "total": 10000,
-                    "namespace": "nimbus-validation-2",
-                    "randomizationUnit": "nimbus_id"
-                },
-                "schemaVersion": "1.5.0",
-                "userFacingName": "Home screen sections test",
-                "referenceBranch": "control",
-                "proposedDuration": 14,
-                "isEnrollmentPaused": false,
-                "proposedEnrollment": 7,
-                "userFacingDescription": "Experiment to test the home screen configurations",
-                "last_modified": 1621443780172
+          "appId": "org.mozilla.ios.Firefox",
+          "appName": "firefox_ios",
+          "application": "org.mozilla.ios.Firefox",
+          "arguments": {},
+          "branches": [
+            {
+              "feature": {
+                "enabled": true,
+                "featureId": "homescreen",
+                "value": {
+                  "sections-enabled": {
+                    "jumpBackIn": true,
+                    "libraryShortcuts": false,
+                    "recentlySaved": true
+                  }
+                }
+              },
+              "ratio": 1,
+              "slug": "control"
+            }
+          ],
+          "bucketConfig": {
+            "count": 10000,
+            "namespace": "ios-homescreen-release-3",
+            "randomizationUnit": "nimbus_id",
+            "start": 0,
+            "total": 10000
+          },
+          "channel": "developer",
```

Comment:
I pulled this experiment from https://experimenter.services.mozilla.com/nimbus/all-users-homescreen-ios-production-100-rollout/details#recipe-json , and this was all I changed.